test/functional/eval/environ_spec.lua

```diff
@@ -17,3 +22,63 @@ describe('environment variables', function()
     eq(0, exists('$DOES_NOT_EXIST'))
   end)
 end)
+
+describe('empty $HOME', function()
+  local original_home = ''
+
+  -- save $HOME before each test
+  before_each(function()
+    original_home = os.getenv('HOME')
+  end)
+
+  -- recover $HOME after each test
+  after_each(function()
+    if original_home ~= nil then
+      setenv('HOME', original_home)
+    end
+    os.remove('test_empty_home')
+  end)
+
+  local function tilde_in_pwd()
+    -- get files in pwd
```

Comment:
"pwd" means "print working directly". It should be `tilde_in_cwd` / CWD ("current working directory").

Reply:
Done